codesigndoc/codesigndoc.go

```diff
@@ -122,3 +108,86 @@ func ExportCodesignFiles(archivePath, outputDirPath string, certificatesOnly boo
 
 	return certsUploaded, provProfilesUploaded, nil
 }
+
+func getFilesToExport(archivePath string, installedCertificates []certificateutil.CertificateInfoModel, installedProfiles []profileutil.ProvisioningProfileInfoModel, certificatesOnly bool) ([]certificateutil.CertificateInfoModel, []profileutil.ProvisioningProfileInfoModel, error) {
+	macOS, err := xcarchive.IsMacOS(archivePath)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var certificate certificateutil.CertificateInfoModel
+	var archive Archive
+	var achiveCodeSignGroup export.CodeSignGroup
+
+	if macOS {
+		archive, achiveCodeSignGroup, err = getMacOSCodeSignGroup(archivePath, installedCertificates)
+		if err != nil {
+			return nil, nil, err
+		}
+		certificate = achiveCodeSignGroup.Certificate()
+	} else {
+		archive, achiveCodeSignGroup, err = getIOSCodeSignGroup(archivePath, installedCertificates)
+		if err != nil {
+			return nil, nil, err
+		}
+		certificate = achiveCodeSignGroup.Certificate()
+	}
+
+	certificatesToExport := []certificateutil.CertificateInfoModel{}
+	profilesToExport := []profileutil.ProvisioningProfileInfoModel{}
+
+	if certificatesOnly {
+		ipaExportCertificate, err := collectIpaExportCertificate(certificate, installedCertificates)
```

Comment:
`collectIpaExportCertificate` -> `collectExportCertificate` or similar (since it is may a macos export)